lib/bambora/json_request.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Bambora
+  class JSONRequest
```

Comment:
This is shared code for all JSON APIs. We'll also need an `XMLRequest` for some of the endpoints.